src/views/guidelines/guidelines.scss

```diff
@@ -1,23 +1,216 @@
 @import "../../colors";
 @import "../../frameless";
 
-.guidelines {
-    .guidelines-footer {
-        margin-top: 1.5rem;
-        text-align: center;
+#view {
+    padding: 0;
+}
+
+.guidelines-page {
+    header {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        background-color: $motion-blue-3;
+        background-image: url('/svgs/guidelines/header_sprinkles_left.svg'), url('/svgs/guidelines/header_sprinkles_right.svg');
+        background-position: left, right;
+        background-repeat: no-repeat;
+        
+        border-top: 50px solid $ui-blue;
```

Comment:
This blue strip looks weird. It's also there in the image in #5584, but I think it was most likely meant as a placeholder for the navigation bar (which was blue when the design was created).

Reply:
Thanks for the warning.  Tomorrow I will run it by Tif Gagnon to see what she would want to do with the strip.

Reply:
We decided to remove the top border, thank you